.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.0
```

Comment:
This is the standard agreed upon by all the different Ruby version managers (RVM, rbenv) on how the Ruby version is set in a particular directory. We technically don't need it. The only reason why its here is that its nice to use the same Ruby version locally as is used on CI.